src/components/UserGroup.tsx

```diff
@@ -22,18 +22,65 @@ const UserImage = styled(UserPic)`
     display: block;
 `;
 
+const StyledCounter = styled.div`
+    padding: 0px 5px;
+    margin: 2px 0px;
+    border-radius: ${radiusL};
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: ${gray4};
+    position: relative;
+    right: 10px;
+`;
+const StyledSmallCircle = styled.div`
+    width: 10px;
+    height: 10px;
+    border-radius: ${radiusXl};
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: ${gray6};
+    position: relative;
+    right: 15px;
+    transform: translateY(8px);
+`;
+
+const StyledText = styled(Text)`
+    padding-bottom: 1px;
```

Comment:
But why?

Reply:
this is necessary to make + in the center